README.md

```diff
@@ -27,9 +27,9 @@ You will use Trello to report your progress on the MVP and any extra features yo
 
```

Comment:
Consider making a Battleplan.md or a Checklist.md to simplify digging through the readme.